### ISSUE TITLE
About the evaluation metrics SS.

### ISSUE THREAD

**Issue by aihao2000** [bytedance/DEADiff#14]
Is removing the content prompt an automatic process?

**Comment by Tianhao-Qi** [comment]
Yes, we give several examples of filtering content to GPT-4 and automatically process all the prompts.
